static/js/shield_study/main.js

```diff
@@ -0,0 +1,15 @@
+
+function trackEvent(event) {
+    var action = (Math.random() >= 0.5);
+    _gaq.push(['_trackEvent', 'opt-in', 'click-' + action]); // Will need to filter on utm_source
+    event.target.innerHTML = "Thank you! Opted In";
+    event.target.setAttribute("onclick","");
+    if (action) {
+      window.location.href = "https://addons.mozilla.org/firefox/downloads/latest/x-screen-performance-study-1/";
```

Comment:
What's the rationale behind randomly sending people to this page some of the time? Should something else happen when action is false?